### ISSUE TITLE
cannot import role assignment in 2.62

### ISSUE THREAD

**Issue by mikemowgli** [terraform-providers/terraform-provider-azurerm#12109]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

```
$ terraform -v
Terraform v0.15.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/azurerm v2.62.0
```

### Affected Resource(s)

* `azurerm_role_assignment`

### Steps to reproduce

```
terraform import module.synapse_sandbox.azurerm_role_assignment.tamers-synapse-lake-perm /subscriptions/123/providers/Microsoft.Authorization/roleAssignments/456
module.synapse_sandbox.azurerm_role_assignment.tamers-synapse-lake-perm: Importing from ID "/subscriptions/123/providers/Microsoft.Authorization/roleAssignments/456"...
module.synapse_sandbox.azurerm_role_assignment.tamers-synapse-lake-perm: Import prepared!
  Prepared azurerm_role_assignment for import
module.synapse_sandbox.azurerm_role_assignment.tamers-synapse-lake-perm: Refreshing state... [id=/subscriptions/123/providers/Microsoft.Authorization/roleAssignments/456]

│ Error: Cannot import non-existent remote object
│
│ While attempting to import an existing object to "module.synapse_sandbox.azurerm_role_assignment.tamers-synapse-lake-perm", the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check
│ that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
```
The same command is successful using v2.61

**Comment by aristosvo** [comment]
Hi @mikemowgli 

I believe this is a duplicate of #12057, at least seven other issues covering this bug are opened the last few days. The PR fixing this problem is under review.

Most information how to solve it can be found under #12060

**Comment by magodo** [comment]
I'm gonna close this issue as it is fixed by #12076, which is delivered in v2.62.1. Please feel free to reopen if the issue remains.
